lib/oban/migrations.ex

```diff
@@ -148,13 +167,20 @@ defmodule Oban.Migrations do
     end
   end
 
-  defp change(prefix, range, direction) do
+  defp change(prefix, range, direction, create_schema \\ true) do
     for index <- range do
       pad_idx = String.pad_leading(to_string(index), 2, "0")
 
+      args =
+        if index == 1 && direction == :up do
+          [prefix, create_schema]
+        else
+          [prefix]
+        end
+
       [__MODULE__, "V#{pad_idx}"]
       |> Module.concat()
-      |> apply(direction, [prefix])
+      |> apply(direction, args)
```

Comment:
What if this were `[opts]`, where `opts` included the prefix and whether to create the schema? That would avoid the variable arity and the special casing for the first migration.

Reply:
Yes I agree. That would require modifying each migration to receive a keyword as argument instead of `prefix`, and that's why I didn't do it. I'll do it now.